assets/js/modules/analytics/common/account-select.test.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Account Select component tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import apiFetchMock from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import AccountSelect from './account-select';
+import { fireEvent, render } from '../../../../../tests/js/test-utils';
+import { STORE_NAME as modulesAnalyticsStoreName, ACCOUNT_CREATE } from '../datastore/constants';
+import * as fixtures from '../datastore/__fixtures__';
+
+// Mock apiFetch so we know if it's called.
+jest.mock( '@wordpress/api-fetch' );
+apiFetchMock.mockImplementation( ( ...args ) => {
+	// eslint-disable-next-line no-console
+	console.warn( 'apiFetch', ...args );
```

Comment:
We wouldn't want to do it project-wide, because this module uses the real implementation in the datastore tests where we mock `fetch` instead. It would be nice if we could use it across all component tests though?

Reply:
We chatted about this a bit in Slack, but for now maybe leave this as-is. It'd be cool if, in the future, we had something we can call that enables/disables that behaviour in tests, just to consolidate the behaviour. And then you could call it in `beforeAll` and `afterAll`, something like:

```js
beforeAll(() => {
  warnOnFetchCall(true);
});

afterAll(() => {
  warnOnFetchCall(false);
});
```

but let's leave it to a follow-up issue. This works for now and is fine 😄